public/app/plugins/datasource/prometheus/datasource.tsx

```diff
@@ -1239,6 +1260,32 @@ export class PrometheusDatasource
   interpolateString(string: string) {
     return this.templateSrv.replace(string, undefined, this.interpolateQueryExpr);
   }
+
+  getDebounceTimeInMilliseconds(): number {
```

Comment:
This might accept an optional parameter named `fallback`. If that is present it returns it in `default` case, otherwise, it returns 350. WDYT?

Reply:
I'm not sure that's necessary, I just removed all the null coalesce that would never get called anyway because this function will always return 350 by default if no cache rules are set.